testflight/suite_test.go

```diff
@@ -136,32 +135,22 @@ var _ = AfterEach(func() {
 })
 
 func downloadFly(atcUrl string) (string, error) {
-	var flyPath string
 	client := concourse.NewClient(atcUrl, http.DefaultClient, false)
 	readCloser, _, err := client.GetCLIReader("amd64", runtime.GOOS)
 	if err != nil {
-		return flyPath, err
-	}
-	cwd, err := os.Getwd()
-	if err != nil {
-		return flyPath, err
-	}
-	flyPath = path.Join(cwd, "fly")
-	os.Remove(flyPath)
-	outFile, err := os.Create(flyPath)
-	if err != nil {
-		return flyPath, err
+		return "", err
 	}
+	outFile, err := ioutil.TempFile("", "fly")
 	_, err = io.Copy(outFile, readCloser)
 	if err != nil {
-		return flyPath, err
+		return "", err
 	}
 	outFile.Close()
-	err = os.Chmod(flyPath, 755)
+	err = os.Chmod(outFile.Name(), 755)
```

Comment:
Really minor nit, but you can call `Chmod` directly on `outFile`:

```suggestion
	err = outFile.Chmod(0755)
```